ami/interactions/environments/actuators/vrchat_osc_discrete_actuator.py

```diff
@@ -8,6 +8,7 @@
 from .base_actuator import BaseActuator
 
 ACTION_CHOICES_PER_CATEGORY = [3, 3, 3, 2, 2]
+ACTION_CHOICES_PER_CATEGORY_MASKED_JUMP_RUN = [3, 3, 3, 1, 1]  # 去年のAMIと行動設定を合わせるため。
```

Comment:
コメントを英語にしたさがある...けど、これくらいだったら別に日本語でも良くない？という迷いがある...
ということで折衷案として両方併記
```suggestion
ACTION_CHOICES_PER_CATEGORY_MASKED_JUMP_RUN = [3, 3, 3, 1, 1]  # 去年のAMIと行動設定を合わせるため。To follow the action settings of last year's AMI. 
```

Reply:
いや，別に英語じゃなくてもいいと思うよ．これくらいはね．